src/bandit-evaluator.ts

```diff
@@ -18,6 +20,9 @@ export interface BanditEvaluation {
 }
 
 export class BanditEvaluator {
+  private assignmentShards = BANDIT_ASSIGNMENT_SHARDS; // We just hard code this for now
```

Comment:
We've decided for now to just have the bandit assignment shards hard-coded in the SDKs as opposed to passing it through as we do with flags or even just using the flag's number of shards.

🗨️ Relevant slack conversation: ["...should this shard count be tied to the Flag.TotalShards field for the given flag..."](https://eppo-group.slack.com/archives/C06N7AA6M5F/p1718315483212449)